testsuite/features/upload_files/salt_event_parser.py

```diff
@@ -5,21 +5,25 @@
 
 # open salt event log and create correct json from it
 errors = []
-item = [] 
+item = []
 f = open('/var/log/rhn/salt-event.log', "r")
 for line in f.readlines():
     item.append(line)
     if re.search(r"^\}", line):
         if '\"result\": false' in "".join(item):
             item[0] = "{"
             errors.append(item)
-        item = [] 
+        item = []
 f.close()
 
 # parse json and find results ending as failed
 failure_count = 0
 for error in errors:
-    j = json.loads("".join(error))
+    try:
+        j = json.loads("".join(error))
+    except ValueError as e:
```

Comment:
If we want to perform a continue in the iteration, I'd move the `except` block at the end, as following code is also using `j` content:

```suggestion
    try:
        j = json.loads("".join(error))
        if not "return" in j:
            break
        for k in j["return"]:
            if not j["return"][k]["result"]:
                failure_count += 1
                print("\n# Failure", failure_count, ", _stamp:", j['_stamp'], json.dumps(j["return"][k], sort_keys=True, indent=4))
    except ValueError as e:
        print("JSON cannot be parsed due {0}".format(e))
        continue
```

Reply:
@srbarrios Change included, thanks ;-)